### ISSUE TITLE
bottom nav and top nav go underneath after playing in player

### ISSUE THREAD

**Issue by oathzed** [flixclusiveorg/Flixclusive#131]
### Steps to reproduce

1. Play a movie
2. Now press back
3. The navigation pill on the bottom will now be underneath the bottom buttons
 And the status icons on the top will be under the back buttons

### Expected behavior

They should not go under other elements of the Ui

### Actual behavior

They go under elements of the UI

### App version

2.2.0

### Android version

Android 14

### Device

Google pixel 7

### Acknowledgements

- [X] I am ***fully confident*** that this issue is related solely to the app and not to any provider.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the **[pre-release version](https://github.com/flixclusiveorg/Flixclusive/releases/tag/pre-release)**.

**Comment by oathzed** [comment]
Seems fixed in the latest. Closing

**Comment by rhenwinch** [comment]
You've created a demo issue of this on Discord, correct?

**Comment by oathzed** [comment]
> You've created a demo issue of this on Discord, correct?

Yes I did, but you can't see it happen in the video because googles screen record doesn't record the status bar or the nav bar
